osmbase.map

```diff
@@ -42,7 +42,16 @@ PROJECTION
    "init=epsg:OSM_SRID"
 END
 
-
+OUTPUTFORMAT
+  NAME "png"
```

Comment:
this should be renamed so as not to overwrite the default "png" driver. e.g. "png_bw"